pyspark/dl/optim/optimizer.py

```diff
@@ -188,6 +188,14 @@ def setvalidation(self, batch_size, val_rdd, trigger, val_method=["Top1Accuracy"
         callBigDlFunc(self.bigdl_type, "setValidation", self.value, batch_size,
                       trigger, val_rdd, val_method)
 
+    def setmodel(self, model):
```

Comment:
but you can see there are set_train_summary, set_val_summary in the same class. And most python classes use the under_score style. Why don't you follow the majority? Please use the underscore style, and change the other two:)

@zhichao-li looks like code styled in python is not consisted.